workflows/differentialabundance.nf

```diff
@@ -119,6 +119,7 @@ include { SHINYNGS_VALIDATEFOMCOMPONENTS as VALIDATOR       } from '../modules/n
 include { DESEQ2_DIFFERENTIAL as DESEQ2_NORM                } from '../modules/nf-core/deseq2/differential/main'
 include { DESEQ2_DIFFERENTIAL                               } from '../modules/nf-core/deseq2/differential/main'
 include { LIMMA_DIFFERENTIAL                                } from '../modules/nf-core/limma/differential/main'
+include { LIMMA_DIFFERENTIAL as LIMMA_RNASEQ                } from '../modules/nf-core/limma/differential/main'
```

Comment:
The limma / limma voom do not run concurrently, so you don't need to import twice